src/core/thread/network_data_leader.cpp

```diff
@@ -260,12 +260,26 @@ bool LeaderBase::IsOnMesh(const Ip6::Address &aAddress) const
 
     while ((prefix = FindNextMatchingPrefix(aAddress, prefix)) != NULL)
     {
-        if (FindBorderRouter(*prefix) == NULL)
+        // check both stable and full Border Router TLVs
```

Comment:
```suggestion
        // check both stable and unstable Border Router TLVs
```
Do we call unstable `full` ?

Reply:
Changed to "temporary", which is the term the Thread Specification uses.